src/modules/forms/plugins/RequestTransformer/HeadersSection/HeadersSection.js

```diff
@@ -19,72 +19,63 @@ const propTypes = {
 };
 
 class HeadersSection extends Component {
-    renderMembers = ({ fields, hint, title }) => (
-        <div>
-            <div className={row()}>
-                <Label>{ title }</Label>
-                <Control
-                    onClick={() => fields.push({})}
-                    icon="add"
-                />
-            </div>
-            <Hint title>{ hint }</Hint>
-            {
-                fields.map((member, index) => {
-                    return (
-                        <Row key={index} col>
-                            <div className={row()}>
-                                <div className={row('item', {'pair': true})}>
-                                    <Field
-                                        name={`${member}.key`}
-                                        type="text"
-                                        component={Input}
-                                    />
-                                </div>
-                                <div className={row('item', {'pair': true})}>
-                                    <Field
-                                        name={`${member}.value`}
-                                        type="text"
-                                        component={Input}
-                                    />
-                                </div>
-                                <div className={row('control')()}>
-                                    <Control
-                                        onClick={() => fields.remove(index)}
-                                        icon="remove"
-                                    />
-                                </div>
-                            </div>
-                        </Row>
-                    );
-                })
-            }
-        </div>
-    )
-
-    renderHeaders = ({ fields, hint, title }) => {
+    renderMembers = ({ fields, hint, title }) => {
         return (
-            <FieldArray
-                name={`${this.props.name}`}
-                title={title}
-                hint={hint}
-                component={this.renderMembers}
-            />
+            <div>
+                <div className={row()}>
+                    <Label>{ title }</Label>
+                    <Control
+                        onClick={() => fields.push({})}
+                        icon="add"
+                    />
+                </div>
+                <Hint title>{ hint }</Hint>
+                {
+                    fields.map((member, index) => {
+                        return (
+                            <Row key={index} col>
+                                <div className={row()}>
+                                    <div className={row('item', {'pair': true})}>
```

Comment:
could this be `row('item', {pair: true})`?

Reply:
just checked it out! it could be like you pointed out 👍